### ISSUE TITLE
Create skeleton placeholders for the jobs list while loading

### ISSUE THREAD

**Issue by djw8605** [unlhcc/HCCGo#237]
Create placeholder similar to this: http://cloudcannon.com/deconstructions/2014/11/15/facebook-content-placeholder-deconstruction.html

**Comment by caseylargent** [comment]
So until the page loads you want this in it's place?

**Comment by djw8605** [comment]
Yes, until the page loads, I would like a placeholder in the jobs list.

**Comment by caseylargent** [comment]
Would this be the job history page? It seems that the new change with the accordion job lists removed the new page
